cypress/integration/login.js

```diff
@@ -8,20 +8,29 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Valentinas Kasteckis"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.user-info__title').contains('Valentinas Kasteckis');
+        cy.get('.main-nav').find('li').should('have.length', 2);
     })
+
+    it('Date should match todays date', function() {
+        var d = new Date();
+        cy.get('.calendar--selected').find('span').get(".calendar__date").contains(d.getDate());

```

Comment:
_.get(".calendar__date")_ is not needed for this to work correctly, .calendar--selected is enough to find correct element

Reply:
I wouldn't agree, if in the later design stage first element wouldn't be ".calendar__date" then this test case would be incorrect.